### PR TITLE
Db/setup

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
- prepare project for connection with mongodb
- set up routing structure for api
- implement db connection in the server.ts
- outsource topichandler into a component

### CODE REVIEW

**Review by Torfeltorfel — APPROVED** [review]
Looks good to me :) 
Maybe some tiny naming issues. 
Besides: If you are in the project, you absolute know what "Needs" are and hence, the naming like "onNeedSubmit" make totally sense. However, as somebody from outside the project, the word "Need" implies that a function might be triggered that should push the user to upvote instead of the fact, that the user just submitted a "Need"

File: src/server/server.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ app.get('*', (_request, response) => {
   response.sendFile('index.html', { root: 'dist/app' })
 })
 
-app.listen(port, () => {
-  console.log(`Server listening on port ${port}!`)
+const url = process.env.MONGODB_API_KEY || ''
+console.log(url)
```
[the remark this answers is not in the document]
Comment (F-Kirchhoff) [reply src/server/server.ts:24]: omg yes!

File: src/app/components/TopicHandler/TopicHandler.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ) => (needId: string) => (updatedVotes: number) => void
+  onNeedSubmit: (topicId: string) => (need: Need) => void
+}
+
+export default function TopicHandler({
+  topics,
+  onNeedUpvote,
+  onNeedSubmit,
+}: TopicHandlerProps): JSX.Element {
+  const params = useParams()
```
Comment (PReichetanz) [inline src/app/components/TopicHandler/TopicHandler.tsx:20]: You can destructure the params as well!
```suggestion
  const { topicId } = useParams()
```
But then you have a naming douplication the line below, how sad :D

**Review by PReichetanz — APPROVED** [review]
Looks good to me!

**Pull request merged**